tests/e2e_tests.py

```diff
@@ -0,0 +1,364 @@
+import os.path
+from models.output import Output
+from generators.model_generator import ModelGenerator
+from send_generated_input_script.rabbit_mq_publisher import RabbitMqPublisher
+from unittest import TestCase
+from tests.test_base import TestBase
+
+
+class E2ETest(TestCase, TestBase):
+
+    def setUp(self):
+        self.rabbitmq_publisher = RabbitMqPublisher()
+        os.chdir(TestBase.STUDENTS_DIR_PATH)
+        if os.path.exists(TestBase.STUDENTS_DIR_PATH):
+            for file in os.listdir(TestBase.STUDENTS_DIR_PATH):
+                os.remove(os.path.join(TestBase.STUDENTS_DIR_PATH, file))
+
+    def test_output_parse(self):
+        input_model = ModelGenerator.generate_input_model()
+        parsed_output = TestBase.parse_output(input_model)
+
+        TestBase.send_body(self.rabbitmq_publisher, input_model)
+
+        output_model = Output(**TestBase.read_from_file(f"{TestBase.STUDENTS_DIR_PATH}/{TestBase.get_docs(1)[0]}"))
+
+        self.assertEqual(output_model.studentDetails, parsed_output.studentDetails.__dict__)
+        for subject in range(len(parsed_output.subjectGrades)):
+            self.assertEqual(output_model.subjectGrades[subject], parsed_output.subjectGrades[subject].__dict__)
+        self.assertEqual(output_model.totalAvg, parsed_output.totalAvg)
+        self.assertEqual(output_model.birthDate, parsed_output.birthDate)
+        self.assertEqual(output_model.age, parsed_output.age)
+        self.assertEqual(output_model.gender, parsed_output.gender)
+        self.assertEqual(output_model.isGoodBehaviour, parsed_output.isGoodBehaviour)
+        self.assertEqual(output_model.age, input_model.age)
+        self.assertEqual(output_model.studentDetails['firstName'], input_model.studentDetails['firstName'])
+        self.assertEqual(output_model.studentDetails['lastName'], input_model.studentDetails['lastName'])
+        self.assertEqual(output_model.studentDetails['id'], input_model.studentDetails['id'])
```

Comment:
why so many assertequals?
cant you connect them?

Reply:
is it ok to create an assert function in the e2e tests class? 
for example: def sanity_test_asserts
and to put inside all of the asserts ?